EventFiltering/PWGCF/CFFilterAll.cxx

```diff
@@ -1161,6 +1275,18 @@ struct CFFilter {
       case CFTrigger::kLambda:
         LOG(fatal) << "No PID selection for Lambdas";
         break;
+      case CFTrigger::kPion:
+        nSigmaIts = track.itsNSigmaPi();
+        if (pThres) {
+          nSigma = nSigmaTPC[2];
+        } else {
+          if (charge > 0) {
+            nSigma = std::sqrt(std::pow(nSigmaTPC[2] - TPCTOFAvg[8], 2) + std::pow(track.tofNSigmaPr() - TPCTOFAvg[9], 2));
+          } else {
+            nSigma = std::sqrt(std::pow(nSigmaTPC[2] - TPCTOFAvg[10], 2) + std::pow(track.tofNSigmaPr() - TPCTOFAvg[11], 2));
```

Comment:
You can use [`RecoDecay::sqrtSumOfSquares`](https://github.com/AliceO2Group/O2Physics/blob/757752c4aa7755285d8e4f0bec1ebb8bb6824a2c/Common/Core/RecoDecay.h#L215) here.